Stratis.Bitcoin/Configuration/TextFileConfiguration.cs

```diff
@@ -44,10 +40,11 @@ public TextFileConfiguration(string[] args)
             foreach (var arg in args)
             {
                 string[] splitted = arg.Split('=');
+                // we must also trim in case the CL commands has spaces: test = testValue1
```

Comment:
This is quite unlikely to happen. It would require someone to write command line as

`program "arg = value"`

instead of 

`program arg=value`

but also instead of 

`program arg = value`

So you kind of assume that the user is familiar with how to properly escape with quotes `"` but you assume the user to be unaware of the fact that there should be no spaces. 

I'm entirely unsure this is something that should be supported. 

On the other hand, however, it probably does no harm, I just can't understand the logic why would you want to support it.

Also see [Comment Policy](https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/commenting-policy.md).SENTENCES rule